ChatApp/ChatApp.API/Mapper/ChatProfile.cs

```diff
@@ -0,0 +1,93 @@
+﻿using AutoMapper;
+using ChatApp.API.ViewModels.Chat;
+using ChatApp.Core.DTO;
+using ChatApp.Core.Entities;
+using ChatApp.CQRS.Commands.Chats;
+using MongoDB.Bson;
+using System;
+using System.Collections.Generic;
+
+namespace ChatApp.API.Mapper
+{
+    public class ChatProfile : Profile
+    {
+        public ChatProfile()
+        {
+            CreateMap<(CreateChatViewModel model, string creator), CreateChatCommand>()
```

Comment:
ConvertUsing is a good method but it's needed when you have 2 completely different entities. Try to refactore in some cases without ConvertUsing usage.